src/App.js

```diff
@@ -65,33 +70,36 @@ function App() {
       jsPsych.data.addProperties({
         participant_id: participantId,
         study_id: studyId,
-        timestamp: timestamp,
         start_date: startDate,
       });
       setLogin(loggedIn)
     },
-    [startDate, timestamp],
+    [startDate],
```

Comment:
startDate isn't state, can this be a normal callback now? also, check `loggedIn` is true before saving jsPsych data

Reply:
startDate still has to be passed in to the callback function apparently